src/main/groovy/org/jetbrains/intellij/IntelliJPlugin.groovy

```diff
@@ -55,16 +63,35 @@ class IntelliJPlugin implements Plugin<Project> {
 
     private static def configurePlugin(@NotNull Project project, @NotNull IntelliJPluginExtension extension) {
         project.afterEvaluate {
-            LOG.info("Preparing IntelliJ IDEA dependency task")
-            configureIntelliJDependency(it, extension)
-            configurePluginDependencies(it, extension)
-            configureInstrumentTask(it, extension)
+            LOG.info("Preparing IntelliJ IDEA tasks")
+            if (extension.configureDependencies) {
+                configureIntelliJDependency(it, extension)
+                configurePluginDependencies(it, extension)
+            }
```

Comment:
Is there ever a reason I would want to run one of these configures without the other?